package.json

```diff
@@ -55,7 +48,6 @@
     "@typescript-eslint/eslint-plugin": "^4.15.1",
     "@typescript-eslint/parser": "^4.15.1",
     "classnames": "^2.2.6",
-    "codecov": "^3.8.1",
```

Comment:
Had to move codecov to the package.json that references it. Otherwise Yarn couldn't find its binary (binaries still get installed in the packages that install them, not the top level necessarily).